rest/utilities_testing.go

```diff
@@ -536,7 +536,8 @@ func (rt *RestTester) GetUserAdminAPI(username string) auth.PrincipalConfig {
 // GetSingleTestDatabaseCollection will return a DatabaseCollection if there is only one. Depending on test environment configuration, it may or may not be the default collection.
 func (rt *RestTester) GetSingleTestDatabaseCollection() (*db.DatabaseCollection, context.Context) {
 	c := db.GetSingleDatabaseCollection(rt.TB(), rt.GetDatabase())
-	return c, c.AddCollectionContext(rt.Context())
+	ctx := base.UserLogCtx(c.AddCollectionContext(rt.Context()), "gotest", base.UserDomainBuiltin, nil)
```

Comment:
The UserLogCtx feels like it is getting set arbitrarily in a lot of places.  Can you help me understand the general approach we're using to decide whether to set "gotest"?  I would have thought a DatabaseCollection shouldn't have a user context (since it's not a DatabaseCollectionWithUser).

Reply:
This is for code that works directly in a test: collection.Put(ctx, ..) needs a user and this provides a user. I wanted to make it clear that this wasn't a real user. This is for cases like import tests where we directly call functions outside of REST calls.